recipes/met4j/meta.yaml

```
@@ -0,0 +1,26 @@
+package: 
+  name: met4j
+  version: 2.0.0
+
+source: 
+  url: https://forgemia.inra.fr/metexplore/met4j/-/package_files/24763/download
+  sha256: b192e70794df6b9320cb0d9219eb53c36a5e27f5da73e3d49a6a65141425918d
+
+build: 
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('met4j', max_pin="x.x.x") }}
```

```suggestion
    - {{ pin_subpackage('met4j', max_pin="x.x.x") }}
```
Version `2.0.0` suggests a stable version.
If SemVer is used then the `max_pin` value should be `"x"` instead

---

Ok :+1: